package.json

```diff
@@ -270,7 +269,11 @@
   "private": true,
   "scripts": {
     "preinstall": "node -e 'process.exit(0)'",
-    "test": "node script/test"
+    "test": "node script/test",
+    "build": "electron-rebuild",
+    "build:apm": "cd apm; yarn install; ./node_modules/atom-package-manager/bin/npm rebuild",
+    "start": "electron .",
+    "dist": "node script/electron-builder.js"
```

Comment:
I'd rather see the electron-builder as a part of the current build process. Then, the duplicate parts of the current build can be removed. See #311 for an example to run this. I did it for Parcel before generating the snapshots.

Reply:
I actually made this experiment to see if we could _replace_ the current build process, as it seems completely unreliable. But I also see your point. Maybe we can discuss this here? https://github.com/atom-community/atom/discussions/443

Reply:
I added my response here:
https://github.com/atom-community/atom/discussions/443#discussioncomment-3111302